recipe/python-cmake-args.patch

```diff
@@ -0,0 +1,45 @@
+From c9c80fe25330f82a9fef33e2826cd0430c864f00 Mon Sep 17 00:00:00 2001
+From: Min RK <benjaminrk@gmail.com>
+Date: Fri, 21 Jun 2024 20:56:58 +0200
+Subject: [PATCH] python: respect CMAKE_ARGS env
+
+---
+ python/setup.py | 9 +++------
+ 1 file changed, 3 insertions(+), 6 deletions(-)
+
+diff --git a/python/setup.py b/python/setup.py
+index d4aa8d38e..95bbd7634 100644
+--- a/python/setup.py
++++ b/python/setup.py
+@@ -1,5 +1,6 @@
+ import os
+ import re
++import shlex
+ import sys
+ import platform
+ import subprocess
+@@ -48,7 +49,8 @@ class CMakeBuild(build_ext):
+ 
+     def build_extension(self, ext):
+         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
+-        cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
++        cmake_args = shlex.split(os.environ.get("CMAKE_ARGS", ""))
```

Comment:
need to pass CMAKE_ARGS to get proper cross compilation, same as https://github.com/FEniCS/dolfinx/pull/2318